sensu-plugins-influxdb-metrics-checker.gemspec

```diff
@@ -3,6 +3,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 require 'date'
 require_relative './lib/sensu-plugins-influxdb-metrics-checker'
+require_relative './lib/time-management'
```

Comment:
I'm not sure that this is necessary, I might be wrong but I think that `require_relative './lib/sensu-plugins-influxdb-metrics-checker'` is needed for the version info in the [gemspec](https://github.com/pliyosenpai/sensu-plugins-influxdb-metrics-checker/blob/master/sensu-plugins-influxdb-metrics-checker.gemspec#L28)